sphinx_source/bezeyiciler.rst

```diff
@@ -0,0 +1,151 @@
+.. meta::
+   :description: Python 3.x'te bezeyiciler (Decoratorlar)
+   :keywords: python, fonksiyonlar, decoratorlar, bezeyiciler
+.. highlight:: py3
+
+*********************************************
+Fonksiyonlarda Bezeyicilerin (Decorator) Kullanımı
+*********************************************
+
+Bu bölüme kadar fonksiyonları nasıl kullanabileceğimizi işlemiştik. 
+Bu yazıda decoratorların nasıl çalıştığını, decoratorların ne olduğunu ve
+ve kendinize özgü decoratorların nasıl oluşturabileceğinizi anlayacağız.
+
+Python'daki herşey objedir, fonksiyonlar dahil. Ama nasıl?
+Pythonda fonksiyonlar first-class'tır, yani bu da demek oluyor ki
+fonksiyonlar bir değişken tarafından referanslanabilir, listelere eklenebilir,
+argumen olarak alınabilir vb. Python'da ki sayılarda birer first-class objedir.
+Yani sayılarda yapabileceğiniz herşeyi (Metodları hariç, çünkü fonksiyon objeleri
+ile sayı objeleri aynı objeler değildir.) yapabilirsiniz. Şimdi örneklere başlayalım.
+
+Fonksiyonlar diğer değişkenler tarafından referanslanabilir
+****
+
+Python'da "Merhaba, dünya!" yazdıran bir fonksiyon örneği::
+   
+   def mrbDunya():
```

Comment:
PEP 8 tarafından açıkça belirtilen bir [fonksiyon isimleri konvensiyonuna](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) uyman gerekiyor. 

`mrbDunya ` => `mrb_dunya` ya da çok daha okunaklı olarak `merhaba_dunya` seklinde yazılmalıdır. (Alttakiler için de geçerli)